content/blog/2024-05-20-summer-software-open-science-school.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Applications now open for a Summer School in Open Science + Research Software Engineering"
+date: 2024-05-20
+author: "Madicken Munk"
+---
+
+
+__Do you want your science, research, and software to be open and accessible? Do you use or develop software in your research? Do you have some basic skills and would like to build on and expand them?__
+
+If this sounds like you, then you might be interested in the upcoming Summer School in Open Science and Research Software Engineering. 
+In July 2024, we will be hosting a five-day workshop on open science and research software engineering at the University of Illinois Urbana-Champaign. 
+
+This workshop complements previous [summer](https://urssi.us/blog/2024/02/29/applications-now-open-for-the-2024-urssi-summer-school-in-research-software-engineering/) and [winter](https://urssi.us/blog/2023/10/31/applications-now-open-for-the-2024-urssi-winter-school-in-research-software-engineering/) schools hosted by URSSI on research software engineering. 
+At this school, students will hone their open science skills in addition to building their skillset in research software engineering. 
+To that end, we strongly encourage attendees to consider a particular software project to apply learning principles to during the school. 
```

Comment:
The existing software lessons sort of rely on people coming with something to work on - so you may want to make bringing a project (or multiple things!) mandatory and not just strongly suggested